src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -153,7 +156,17 @@ export default {
 		loans() {
 			if (this.loans.length) {
 				this.$nextTick(() => {
-					this.$refs.campaignLoanCarousel.reInit();
+					if (this.$refs.campaignLoanCarousel) {
+						// re-init carousel
+						this.$refs.campaignLoanCarousel.reInit();
+						// shake the carousel to re-init controls
```

Comment:
I admit this is a janky way to handle things but it works well enough for now. With the new loading sequence, it seems that the carousel controls weren't re-initialized properly. Specifically, the "next" arrow remained disabled after re-init. It would not activate on resizing the page but would re-activate if you swiped to scroll one loan over... I verified all state before and after the re-init but it seemed like the internal checks on the button weren't being updated.